docs/uberon-editor-sop.md

```diff
@@ -97,8 +97,8 @@ In cases where more than one CURIE is available for a resource, either is accept
 
 For ORCIDs: Enter the ORCID as an IRI in the IRI field on the "IRI Editor" tab, for example `https://orcid.org/0000-0002-7356-1779`.
 
-For URLs: In the "Create Annotation" window, select the annotation property **seeAlso** (`rdfs:seeAlso`). Enter the URL as a literal string with Datatype `xsd:anyURI` selected.
-DO NOT use database_cross_reference with a value that is a URL. 
+For URLs: In the "Create Annotation" window, select the annotation property **seeAlso** (`rdfs:seeAlso`). Enter the URL as a literal string. DO NOT use database_cross_reference with a value that is a URL. Note: In OWL-based files (like cl-edit.owl), Datatype `xsd:anyURI` is also selected; however, in OBO-based files (like uberon-edit.obo) these always become strings, so no selection needs to be made for Datatype in Uberon. Datatype selection is planned to be implemented in a future OBO revision, and updates can be checked at https://github.com/owlcollab/oboformat/issues/128.
```

Comment:
@matentzn, can you kindly confirm which "see also" annotation property should be used? There are currently three  of which two may be confused with rdfs:seeAlso.

**'see also'** http://www.w3.org/2000/01/rdf-schema#seeAlso
**seeAlso** http://www.geneontology.org/formats/oboInOwl#seeAlso
**seeAlso** http://www.geneontology.org/formats/oboInOwl#http://www.w3.org/2000/01/rdf-schema#seeAlso

And does the same apply to CL?
Thank you.

Reply:
It should be only the first one.

The last one should be removed.

Reply:
related issue: https://github.com/owlcollab/oboformat/issues/143